# Göran Uddeborg <goeran@uddeborg.se>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-04-07 14:44+0200\n"
"PO-Revision-Date: 2017-03-27 12:00-0400\n"
"Last-Translator: Göran Uddeborg <goeran@uddeborg.se>\n"
"Language-Team: Swedish\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: sv\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 3.9.6\n"

#: ../plugins/kickstart.py:58
msgid "Install packages defined in a kickstart file on your system"
msgstr "Installera paket definierade i en kickstart-fil på ditt system"

#: ../plugins/kickstart.py:63
msgid "kickstart file"
msgstr "kickstart-fil"

#: ../plugins/kickstart.py:81
#, python-format
msgid "file cannot be parsed: %s"
msgstr "filen kan inte tolkas: %s"

#: ../plugins/kickstart.py:98
#, python-format
msgid "No package %s available."
msgstr "Inget paket %s är tillgängligt."

#: ../plugins/kickstart.py:103
msgid "Nothing to do."
msgstr "Inget att göra."

#: ../plugins/rpm_conf.py:67
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "Lägger till ”{}” till listan av hanterade paket för rpmconf"

#: ../plugins/rpm_conf.py:73
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr "insticksmodulen rpmconf kommer inte köra i icke-interaktivt läge"

#: ../plugins/rpm_conf.py:86
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr "ignorerar sys.exit från rpmconf på grund av saknad MERGE-variabel"

#: ../plugins/rpm_conf.py:90
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr "ingorerar sys.exit från rpmconf på grund av saknad fil"

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "anslutning till snapperd misslyckades: %s"

#: ../plugins/snapper.py:54
msgid "creating snapshot"
msgstr "skapar ögonblicksbild"

#: ../plugins/snapper.py:59
#, python-format
msgid "created snapshot %d"
msgstr "skapade ögonblicsbild %d"

#: ../plugins/snapper.py:63
#, python-format
msgid "creating snapshot failed: %s"
msgstr "att skapa en ögonblicksbild misslyckades: %s"

#. Translators: This string is only used in unit tests.
#: ../plugins/system_upgrade.py:54
msgid "the color of the sky"
msgstr "himlens färg"

#: ../plugins/system_upgrade.py:73
msgid "No new kernel packages were found."
msgstr "Inga nya kärnpaket hittades."

#: ../plugins/system_upgrade.py:75
msgid "Need a --releasever greater than the current system version."
msgstr "Behöver en --releasever som är högre än den aktuella systemversionen."

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:77
msgid ""
"Download complete! Use 'dnf system-upgrade reboot' to start the upgrade."
msgstr ""
"Hämtningen färdig!  använd ”dnf system-upgrade reboot” för att starta "
"uppgraderingen."

#: ../plugins/system_upgrade.py:79
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""
"Tyvärr, du behöver använda ”download --releasever” istället för ”--network”"

#: ../plugins/system_upgrade.py:112
msgid "--datadir: File exists"
msgstr "--datadir: Filen finns"

#: ../plugins/system_upgrade.py:257
msgid "The following boots appear to contain upgrade logs:"
msgstr "Följande uppstarter verkar innehålla uppgraderingsloggar:"

#: ../plugins/system_upgrade.py:267
msgid "-- no logs were found --"
msgstr "— inga loggar hittades —"

#: ../plugins/system_upgrade.py:282
msgid "Cannot find logs with this index."
msgstr "Kan inte hitta loggar med detta index."

#: ../plugins/system_upgrade.py:305
msgid "Prepare system for upgrade to a new release"
msgstr "Förbered systemet för uppgradering till en ny utgåva"

#: ../plugins/system_upgrade.py:314
msgid "save downloaded data to this location"
msgstr "spara hämtad data på denna plats"

#: ../plugins/system_upgrade.py:317
msgid "keep installed packages if the new release's version is older"
msgstr "behåll installerade paket om den nya utgåvans version är äldre"

#: ../plugins/system_upgrade.py:398
msgid "system is not ready for upgrade"
msgstr "systemet är inte redo för en uppgradering"

#: ../plugins/system_upgrade.py:400
msgid "upgrade is already scheduled"
msgstr "uppgraderingen är redan schemalagd"

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:406
msgid "use 'dnf system-upgrade reboot' to begin the upgrade"
msgstr "använd ”dnf system-upgrade reboot” för att påbörja uppgraderingen"

#: ../plugins/system_upgrade.py:408
msgid "another upgrade tool is running. exiting quietly."
msgstr "ett annat uppgraderingsverktyg kör.  avslutar tyst."

#: ../plugins/system_upgrade.py:432
msgid "Rebooting to perform upgrade."
msgstr "Startar om för att utföra uppgraderingen."

#: ../plugins/system_upgrade.py:459 ../plugins/system_upgrade.py:465
msgid "Starting system upgrade. This will take a while."
msgstr "Påbörjar systemuppgraderingen.  Detta kommer ta en stund."

#: ../plugins/system_upgrade.py:496
msgid "Cleaning up downloaded data..."
msgstr "Rensar upp hämtade data …"

#: ../plugins/system_upgrade.py:527
msgid "Download finished."
msgstr "Hämtningen klar."

#: ../plugins/system_upgrade.py:531 ../plugins/system_upgrade.py:532
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "Uppgraderingen är klar!  Rensar upp och startar om …"

#: ../plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""
"Avaktiverar insticksmodulen torproxy: kan inte anslute till Tor-nätverket"

#: ../plugins/tracer.py:75
msgid "For more information run:"
msgstr "För mer information kör:"

#: ../plugins/tracer.py:102
msgid "Call to Tracer API ended unexpectedly:"
msgstr "Anrop till spårnings-API:et avslutades oväntat:"

#: ../plugins/tracer.py:104
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""
"Besök https://github.com/FrostyX/tracer/issues och skicka in problemet.  "
"Tack"

#: ../plugins/tracer.py:106
msgid "We apologize for any inconvenience"
msgstr "Vi ber om ursäkt för eventuella besvär"
